src/actions/actionHistory.tsx

```diff
@@ -58,7 +58,9 @@ const writeData = (
   return { commitToHistory };
 };
 
-type ActionCreator = (history: History) => Action;
+type ActionCreator = (
+  history: Pick<History, "undoOnce" | "redoOnce">,
```

Comment:
I've limitted `History` type here in order to comply with limitted `history` API, e.g. `createRedoAction(excalidrawAPI?.history)`